src/xdpinstaller/Product.wxs

```diff
@@ -6,105 +6,86 @@ SPDX-License-Identifier: MIT
 <?define ProductVersion="{9363C0E3-4DE9-4067-9F5E-6A1A06034B59}"?>
 <?define UpgradeCode="{79F93392-843E-4B85-824B-2CFC7D16F080}"?>
 <Wix xmlns="http://schemas.microsoft.com/wix/2006/wi" xmlns:ui="http://schemas.microsoft.com/wix/UIExtension" xmlns:util="http://schemas.microsoft.com/wix/UtilExtension">
-	<Product Id="$(var.ProductVersion)" Name="XDP for Windows" Language="1033" Version="0.16.9" Manufacturer="Microsoft" UpgradeCode="$(var.UpgradeCode)">
-		<Package Description="XDP for Windows" InstallerVersion="301" Compressed="yes" InstallScope="perMachine" Manufacturer="Microsoft" Platform="x64" />
-		<MajorUpgrade AllowSameVersionUpgrades="yes"
-					  Disallow="yes" DisallowUpgradeErrorMessage="An older version of [ProductName] is already installed. Please remove it first."
-					  AllowDowngrades="no" DowngradeErrorMessage="A newer version of [ProductName] is already installed. Please remove it first." Schedule="afterInstallFinalize" />
-		<MediaTemplate EmbedCab="yes" />
+    <Product Id="$(var.ProductVersion)" Name="XDP for Windows" Language="1033" Version="$(var.XdpMajorVersion).$(var.XdpMinorVersion).$(var.XdpPatchVersion)" Manufacturer="Microsoft" UpgradeCode="$(var.UpgradeCode)">
+        <Package Description="XDP for Windows" InstallerVersion="301" Compressed="yes" InstallScope="perMachine" Manufacturer="Microsoft" Platform="x64" />
+        <MajorUpgrade AllowSameVersionUpgrades="no"
```

Comment:
A thought I had: Could we support installing major versions of XDP side-by-side (different IOCTL/service/binary names maybe)? Would we ever want to?

Reply:
Right, yeah, that's something I had been pondering, but this is just to get a barebones MSI in and integrated.

Long run, yes, being able to have multiple MSIs installed concurrently could be useful. I'm not sure we'd want multiple versions of XDP all handling IOCTLs, inspecting packets, etc. at the same time.